.github/workflows/release.yml

```diff
@@ -37,6 +37,11 @@ jobs:
           GIT_COMMITTER_NAME: asyncapi-bot
           GIT_COMMITTER_EMAIL: info@asyncapi.io
         run: npm run release
+      # Sleep for 10 seconds before using latest version in playground, because sometimes NPM needs additional few seconds to `save` package in registry
+      - name: Sleep for 10 seconds
+        uses: jakejarvis/wait-action@master
```

Comment:
please use something different, this action looks nifty but as you can see it is always fetched from master 😱 which is super unsecure. Unfortunately the user doesn't seem to provide regular professional releases

why not just `sleep 30s && npm install @kyma-project/asyncapi-react@${{ steps.extractver.outputs.version }} -s` in the playground step?

Reply:
Yes, you are right, I changed it to `sleep 10 && ...`